lib/link_web/controllers/dashboard_controller.ex

```diff
@@ -15,13 +15,17 @@ defmodule LinkWeb.DashboardController do
       |> Enum.into(MapSet.new())
 
     available_studies = Studies.list_studies(exclude: exclusion_list)
+    available_count = Enum.count(available_studies)
 
-    selected_studies = [{"owned", owned_studies}, {"participations", study_participations}]
+    highlighted_studies = [{"owned", owned_studies}, {"participations", study_participations}]
+    highlighted_count = Enum.count(exclusion_list)
```

Comment:
This number could be off with the total number of studies shown as highlighted. This happens if a user is both a participant and an owner (not very likely?). Then the exclusion list will have this study ID only once (since it is a set).

Reply:
hmm interesting. I personally think we need to prevent to be participant on ones own study. What do you think?